operator/pkg/applier/ssa.go

```diff
@@ -80,10 +80,12 @@ func (s *SetApplier) Delete(deployInfo manifestTypes.InstallInfo, objects *manif
 	deletionSuccess := true
 	deleteErrors := make([]error, 0)
 
-	for _, obj := range objects.Items {
-		name := obj.GetName()
+	for i := len(objects.Items) - 1; i >= 0; i-- {
+		// reverse order of install will ensure that CRDs last applied are uninstalled first
```

Comment:
Wouldnt the CRD installation get stuck if there is still a CR present? Also wouldnt the applier have to work order-independant? If you call it from kustomize, you cannot make order assumptions